rustls-post-quantum/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rustls-post-quantum"
-version = "0.3.0"
+version = "0.2.0"
```

Comment:
What was the story here? It was bumped in https://github.com/rustls/rustls/pull/2259 unnecessarily?

Reply:
Yes, I bumped it to 0.2.0 when moving from kyber to ml-kem, but didn't release it. Then some time passed and I bumped it again to 0.3.0.

Reply:
makes sense, thanks!